app/services/reaction_handler.rb

```diff
@@ -37,14 +37,14 @@ def initialize(params, current_user:)
   delegate :rate_limiter, to: :current_user
 
   def create
-    destroy_contradictory_mod_reactions if reactable_type == "Article"
+    destroy_contradictory_mod_reactions if %w[Article Comment].include?(reactable_type)
```

Comment:
Question - does it (not) make sense to extend this to comments as well? I'm not sure why it was only applied to posts before